lib/main.js

```diff
@@ -278,6 +284,8 @@ export default {
         };
         if (disableExecuteTimeout) {
           execOptions.timeout = Infinity;
+        } else if (executeTimeout) {
+          execOptions.timeout = executeTimeout;
```

Comment:
Just set this up in the initial execOptions declaration, it will _always_ have a value due to this being observed.